cmd/jujud/agent/machine.go

```diff
@@ -1208,6 +1208,7 @@ func (a *MachineAgent) startModelWorkers(controllerUUID, modelUUID string) (work
 		CharmRevisionUpdateInterval: 24 * time.Hour,
 		InstPollerAggregationDelay:  3 * time.Second,
 		StatusHistoryPrunerInterval: 5 * time.Minute,
+		ActionPrunerInterval:        2 * 7 * 24 * time.Hour, // once every a fortnight
```

Comment:
This should be more frequent - if someone changes the config because they only want to keep a day of action results, they're going to be annoyed that it grows for two weeks and then is pruned back to a day. Maybe make it daily?